trezorlib/tests/device_tests/test_msg_cardano_sign_transaction.py

```diff
@@ -113,12 +114,93 @@
     ),
 ]
 
+INVALID_VECTORS = [
+    # Output address is a valid CBOR but invalid Cardano address
+    (
+        # protocol magic (mainnet)
+        764824073,
+        # inputs
+        [
```

Comment:
Can we extract common inputs to some constant so they don't pollute testcase?

Reply:
fixed